ocaml/idl/datamodel_host.ml

```diff
@@ -1364,6 +1364,24 @@ let host_query_ha = call ~flags:[`Session]
     ~hide_from_docs:true
     ()
 
+let set_https_only =
+  call ~name:"set_https_only"
+    ~doc:
+      "updates the host firewall to open or close port 80 depending on the \
+       value"
+    ~lifecycle:[]
```

Comment:
This can't be empty, the lifecycle is not being filled by the build system, I think this is similar to what Ben did when doing the pool secret rotation after release